### ISSUE TITLE
Settings for "Pending state"

### ISSUE THREAD

**Issue by rkaldung** [zammad/zammad#143]
*Note: This post has been updated by @MrGeneration to meet our requirenments.*

### Infos:
* Used Zammad version: 3.0
* Installation method (source, package, ..): any
* Operating system: any
* Database + version: any
* Elasticsearch version: any
* Browser + version: any
* Ticket-ID: 1048181, 1078749

### Desired behavior:

1. System configuration setting for each 'pending state' as an initial time offset added to now() when the state is choosen in the UI. Sth. like 'next working day at 8am'
2. If allowed (system setting) agents are allowed to configure there own time offset.

**Comment by raphaelm** [comment]
Please regard this comment as a polite and respectful bump of this request. :) OTRS just has a default pending time of 24h, it would be nice to at least have a sensible default even without a configurable setting. It would save us a lot of clicks ;)

**Comment by pinscher** [comment]
I think this is the same request #864

**Comment by MrGeneration** [comment]
As I understand this issue, it's about adding a default pending time that's automatically filled when choosing a pending state. The default time should be a system setting which agents can adjust if needed (for themselves)

So no duplicate. :) But thanks for checking!

**Comment by raphaelm** [comment]
It's not the *same* feature that is requested, but if #864 is implemented, this would be much less important at least

**Comment by pinscher** [comment]
I agree to raphaelm. if #864 is implemented you can easily build your prefered "pending timeout". 
In order to make it more user friendly, you can create a default automation that is existing after installation that is re-opening the pending state after 24 hours for example. This is exactly how Zendesk is doing after creating a new zendesk account. Then the user can adjust this predefined automation or disable it.
